src/Cassandra/ICluster.cs

```diff
@@ -98,5 +98,10 @@ public interface ICluster : IDisposable
         ///  used by it. <p> This method has no effect if the cluster was already shutdown.</p>
         /// </summary>
         void Shutdown(int timeoutMs = System.Threading.Timeout.Infinite);
+
+        /// <summary>
+        /// Gets a collection of execution profiles that were configured for this Cluster instance.
+        /// </summary>
+        IReadOnlyDictionary<string, IExecutionProfile> ExecutionProfiles { get; }
```

Comment:
Shouldn't we expose it in the `Configuration` and avoid exposing a new prop on `Cluster` / `Session`, wdyt?

Reply:
I honestly don't remember why I didn't just do that considering that the `Cluster` property is just a wrapper around the `Configuration` one. Done!